packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

```diff
@@ -40,14 +40,11 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/${apiVersion}/radius`<br>Eligible for workforce multiplier                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`<br>Eligible for dynamic scale and workforce multiplier                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
-| <ApiLifecycle access="ie" /> **Identity Engine APIs:**                                                                            |               |               |      |         |                    |
-| `/idp/idx`                                                                                                     | 100              | 600              | 600     | 600        | 600                   |
-| `/idp/idx/identify`<br>Eligible for dynamic scale and workforce multiplier                                                                                                     | 100              | 600              | 600     | 600        | 600                   |
-| `/idp/idx/introspect`<br>Eligible for dynamic scale and workforce multiplier                                                                                                     | 150              | 900              | 900     | 900        | 900                   |
-| `/idp/idx/challenge/poll`                                                                                                     | 100              | 600              | 600     | 600        | 600                   |
-| `/idp/idx/authenticators/poll`                                                                                                     | 100              | 600              | 600     | 600        | 600                   |
-| `/idp/idx/poll`                                                                                                     | 100              | 600              | 600     | 600        | 600                   |
-| Identity Engine App Intent<br>Eligible for dynamic scale and workforce multiplier                                                                                                     | 150              | 900              | 900     | 900        | 900                   |
+| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br>The below rate limits are configured so that orgs are allowed to complete 1000 Identity Engine Authentication flows per minute.                                                                             |               |               |      |         |                    |
```

Comment:
```suggestion
| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br>The below rate limits are configured so that orgs are allowed to complete 1000 Identity Engine authentication flows per minute.                                                                             |               |               |      |         |                    |
```